src/settings-ui/Settings.UI/Controls/ShortcutControl/ShortcutDialogContentControl.xaml

```diff
@@ -2,92 +2,100 @@
-                    CornerRadius="{ThemeResource ControlCornerRadius}"
                     BorderBrush="{ThemeResource InfoBarBorderBrush}"
                     BorderThickness="{ThemeResource InfoBarBorderThickness}"
-                    Padding="8"
-                    Margin="-2,0,0,0"
+                    CornerRadius="{ThemeResource ControlCornerRadius}"
                     Visibility="{Binding ElementName=ShortcutContentControl, Path=IsError, Mode=OneWay, Converter={StaticResource boolToVisibilityConverter}}">
-                <Grid>
-                    <Grid.ColumnDefinitions>
-                        <ColumnDefinition Width="Auto"/>
-                        <ColumnDefinition Width="*" />
-                    </Grid.ColumnDefinitions>
+                    <Grid>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="Auto" />
+                            <ColumnDefinition Width="*" />
+                        </Grid.ColumnDefinitions>
 
-                    <!--TODO(stefan) InfoBadge not available
+                        <!--TODO(stefan) InfoBadge not available
                         <InfoBadge AutomationProperties.AccessibilityView="Raw"
                                         Margin="2,0,12,0"
                                         Style="{StaticResource CriticalIconInfoBadgeStyle}" />
-                    -->
+                        -->
 
-                    <TextBlock x:Name="InvalidShortcutWarningLabel"
-                               x:Uid="InvalidShortcut"
-                               VerticalAlignment="Center"
-                               Margin="0,-1,0,0"
-                               Foreground="{ThemeResource InfoBarTitleForeground}"
-                               FontWeight="{ThemeResource InfoBarTitleFontWeight}"
-                               Grid.Column="1" />
-                </Grid>
-            </Border>
+                        <TextBlock
+                            x:Name="InvalidShortcutWarningLabel"
+                            x:Uid="InvalidShortcut"
+                            Grid.Column="1"
+                            Margin="0,-1,0,0"
+                            VerticalAlignment="Center"
+                            FontWeight="{ThemeResource InfoBarTitleFontWeight}"
+                            Foreground="{ThemeResource InfoBarTitleForeground}" />
+                    </Grid>
+                </Border>
             </Grid>
-            <toolkitcontrols:MarkdownTextBlock x:Uid="InvalidShortcutWarningLabel"
-                                               FontSize="12"
-                                               Background="Transparent"
-                                               Foreground="{ThemeResource TextFillColorSecondaryBrush}"/>
+            <toolkitcontrols:MarkdownTextBlock
+                x:Uid="InvalidShortcutWarningLabel"
+                Background="Transparent"
+                FontSize="12"
```

Comment:
Would it be good to use more [theme resources for fonts](https://learn.microsoft.com/windows/apps/design/style/xaml-theme-resources#the-xaml-type-ramp)? Like, this would be `caption`.